services/movie.py

```diff
@@ -0,0 +1,43 @@
+from db.models import Movie
+from django.db.models import QuerySet
+from typing import Optional
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None
+) -> QuerySet:
+
+    queryset = Movie.objects.all()
+    if None not in (genres_ids, actors_ids):
```

Comment:
why do you need it?

Reply:
if genres_ids:
        queryset = queryset.filter(genres__id__in=genres_ids)

    if actors_ids:
        queryset = queryset.filter(actors__id__in=actors_ids)

Reply:
use like this